src/test/java/hw1/BaseClass.java

```diff
@@ -1,19 +0,0 @@
-package hw1;
```

Comment:
Why do you delete files of the HW1 from this pull request?

Reply:
Just for the principle that only new homework is for a new pull request